lib/moya/db/migrate/20141208231338_add_drds_with_binary_uuids.rb

```diff
@@ -1,3 +1,6 @@
+#TODO Either contribute to activeuuid, or remove it
```

Comment:
It seems a little finicky in Rails 4, I haven't dug into exactly why, but I'd like to start by not needing this require in a migration, and fixing the find call to take a string uuid for starters.